internal/config/0_common_config.go

```diff
@@ -26,6 +26,10 @@ func init() {
 	parser.Viper.SetDefault("database.user", "")
 	parser.Viper.SetDefault("database.password", "")
 	parser.Viper.SetDefault("database.loglevel", "none")
+	parser.Viper.SetDefault("database.minConn", 2)
+	parser.Viper.SetDefault("database.maxConn", 100)
+	parser.Viper.SetDefault("database.maxLifetime", 2*time.Hour)
+	parser.Viper.SetDefault("database.maxIdleTime", 15*time.Minute)
```

Comment:
FYI the default values from the pgx driver are:

* Minimum conn: 0
* Max conn: 4
* Lifetime: 1 hour
* Idletime: 30 minutes

Our values are wild guesses, we need to talk with SRE about this in more detail.